src/main/java/com/epam/handler/GlobalExceptionHandlerController.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.handler;
+
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@ControllerAdvice
+@EnableWebMvc
+public class GlobalExceptionHandlerController {
+
+    private static final String DEFAULT_ERROR_VIEW = "error";
+
+    @ExceptionHandler(Exception.class)
```

Comment:
Please fix this with Throwable at this moment

Reply:
Done